### DISCUSSION TITLE
Container Queries

### DISCUSSION THREAD

**Discussion by Code-Victor** [ora-ui/ora-ui#67] [Ideas]
I think there should be support for [container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries). It's really almost production ready and Tailwindcss already supports it.
![image](https://user-images.githubusercontent.com/63613078/214280828-ca9a1aa5-7e6c-497f-b96f-e188c48daaf4.png)

**Comment by gregogun** [comment]
Agreed, this would be great to have. 

Regarding support, what is exactly needed here for it to work? As Stitches supports the at-rule, my assumption is that it should work out of the box no?

**Comment by Code-Victor** [comment]
It's so shocking that there isn't a way to do it in stitches yet. Stitches only support `@media` queries and not `@container` in the `stitches.config.ts`
I thought of using a `util` function with regular CSS, but it wouldn't be effective while dealing with responsive variants.

I'll start a discussion on the Stitches GitHub repository concerning it.

**Reply by gregogun** [reply]
Oh right I see, yeah that's tricky. 

Ok great, really hope they can add support for this.
